docs/apache-airflow-providers-amazon/operators/redshift_data.rst

```diff
@@ -0,0 +1,52 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+.. _howto/operator:RedshiftDataOperator:
+
+RedshiftDataOperator
+====================
+
+.. contents::
+  :depth: 1
+  :local:
+
+Overview
+--------
+
+Use the :class:`RedshiftDataOperator <airflow.providers.amazon.aws.operators.redshift_data>` to execute
+statements against an Amazon Redshift cluster.
+
+This differs from RedshiftSQLOperator in that it allows users to query and retrieve data via the AWS API and avoid the necessity of a Postgres connection.
```

Comment:
Is this not necessarily true after version 2.4.0 of the Amazon provider is released? Looks like the Postgres dependency is being removed as part of #18844.

Reply:
> Is this not necessarily true after version 2.4.0 of the Amazon provider is released? Looks like the Postgres dependency is being removed as part of #18844.

This is referring to the Redshift requirement that a Postgres URL and port is open to Airflow.  With boto3 redshift-data, no such access need exist

Reply:
Cool, just wanted to confirm. Thanks!